package.json

```diff
@@ -60,7 +60,7 @@
   },
   "packageManager": "yarn@3.2.1",
   "engines": {
-    "node": ">=14.0.0"
+    "node": "16.20.1 || ^18.16.1 || >=20"
```

Comment:
Elsewhere we just use `>=16`. Is there a reason to be so specific with the supported version ranges?

Reply:
Ah I see, fair enough. I ask because it differs from our template. Perhaps we should migrate this change there as well?

RE: the lack of support for v16 below v16.20, I'm guessing that's just an attempt to disregard older versions as well, and associated bugs? Or is there another reason for that minimum, e.g. a feature we rely on.